docs/api.md

```diff
@@ -2019,6 +2100,8 @@ This endpoint supports retrieving extended metadata; provide a header `X-Extende
 
 Currently, the only updatable _metadata_ on a Submission is its `reviewState`. To update the submission _data_ itself, please see [Updating Submission data](/reference/submissions/submissions/updating-submission-data).
 
+Starting with Version 2022.3, changing `reviewState` of a Submission to `approved` can create an Entity in a Dataset if corresponding Form defines the Dataset schema. If an Entity is created successfully then `entity.create` event is logged in Audit logs else `entity.create.error` is logged.
```

Comment:
Small changes to add more "the"s to the sentence.

`Starting with Version 2022.3, changing the `reviewState` of a Submission to `approved` can create an Entity in a Dataset if the corresponding Form defines the Dataset schema. If an Entity is created successfully then an `entity.create` event is logged in Audit logs, else `entity.create.error` is logged.

Reply:
Resolved in https://github.com/sadiqkhoja/central-backend/pull/1